pallets/did/src/tests.rs

```diff
@@ -655,13 +673,13 @@ fn check_max_limit_public_keys_to_remove_did_update() {
 	let auth_key = get_sr25519_authentication_key(true);
 	let alice_did = get_did_identifier_from_sr25519_key(auth_key.public());
 
-	let old_did_details = generate_base_did_details(did::DidVerificationKey::from(auth_key.public()));
+	let old_did_details = generate_base_did_details::<Test>(did::DidVerificationKey::from(auth_key.public()));
 
 	// Max keys allowed + 1
 	let keys_ids_to_remove =
-		get_public_keys_to_remove(<Test as did::Config>::MaxNewKeyAgreementKeys::get().saturating_add(1));
+		get_public_keys::<Test>(<Test as did::Config>::MaxNewKeyAgreementKeys::get().saturating_add(1));
```

Comment:
Ok my bad here as well 😅

```suggestion
		get_public_keys::<Test>(<Test as did::Config>::MaxVerificationKeysToRevoke::get().saturating_add(1));
```